Misc BGP Control Plane documents (#31670)
cilium-team/cilium: Documentation/network/bgp-control-plane-operation.rst

This document provides guidance on how to operate the BGP Control Plane.

Logs
====

> You can check the BGP Control Plane logs from the Cilium agent logs. The logs

Reviewer: ```suggestion
BGP Control Plane logs can be found in the Cilium agent logs. The logs
```

> are tagged with ``subsys=bgp-control-plane``. You can filter the logs like the
> following:

Reviewer: ```suggestion
are tagged with ``subsys=bgp-control-plane``. You can use this tag to filter
the logs as in the following example:
```
